packages/ppx_mica/ppx_mica.0.1.0/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "PPX deriver that automates differential testing for OCaml modules"
+maintainer: ["Ernest Ng <ernest@cs.cornell.edu>"]
+authors: ["Ernest Ng <ernest@cs.cornell.edu>"]
+license: "ISC"
+homepage: "https://github.com/ngernest/mica"
+bug-reports: "https://github.com/ngernest/mica/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "ocaml" {>= "4.13"}
+  "odoc" {with-doc}
+  "core" {>= "v0.15.1"}
+  "ppx_jane" {>= "v0.15.0"}
+  "ppx_assert" {>= "v0.15.0"}
+  "base" {>= "v0.15.1"}
+  "base_quickcheck" {>= "v0.15.0"}
```

Comment:
[ppx_mica.0.1.0 (failed: Library "tyche_utils" not found.)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/51b56fad1033fa4ad04b706eef5074af6f223397/variant/compilers,4.13,ppx_mica.0.1.0) tells us that we are missing the dependency on the `tyche_utils` here:

```suggestion
  "base_quickcheck" {>= "v0.15.0"}
  "tyche_utils" {=version}
```

This uses the [version package variable](https://opam.ocaml.org/doc/Manual.html#pkgvar-version) to ensure the versions stay in sync. If you expect the API of the utils to be stable accross versions, you could probably omit this constraint.

Reply:
Thanks! For simplicity, I've moved the `tyche_utils` package to a separate repo as it's not strictly necessary for the main `ppx_mica` package.